src/components/modals/AddRoomModal.tsx

```diff
@@ -90,6 +95,42 @@ export default function AddRoomModal({
 
     const createSubmitLabel = privacySetting.value === PrivacySetting.TokenGated ? 'Next' : 'Create'
 
+    const [isTokenGatedRoom, setIsTokenGatedRoom] = useState<boolean>(false)
```

Comment:
Yep, this is different. All the gate parts went into the seconds dialog recently (→ `AddTokenGatedRoomModal`) and yet somehow are back. Tricky conflicts!

Do you want me to reshape it back to the recent 2 model flow, and do you see any obstacles?